packages/core/src/render3/instructions/shared.ts

```diff
@@ -266,6 +268,20 @@ export function createTNodeAtIndex(
   return tNode;
 }
 
+/**
+ * Checks if the component of a given view is a standalone one or not.
+ *
+ * @param lView The `LView` containing the view's data
+ * @returns `true` if the component owning this view is a standalone one, `false` otherwise
```

Comment:
I'd suggest that we drop `@returns` as it's covered in the function description.
```suggestion
 * @param lView An `LView` that represents a current component that is being rendered.
```